VERSION

```diff
@@ -1 +1 @@
-v0.9.1.47
+v9.9.9
```

Comment:
Woah, big jump! How are these versions determined? / Does it matter?

Reply:
@omarabed15 During last Wednesday's PR Party we discussed worked flow. We decided to on the convention of
giving feature branches versions numbers of all 9's until a proper version could be assigned. Our CI pipeline includes a check for a valid version number that must be numbers after the "v". So `v9.9.9` is just for the version.  Good notice, however.